source/common/upstream/cluster_manager_impl.cc

```diff
@@ -479,6 +479,37 @@ bool ClusterManagerImpl::addOrUpdateCluster(const envoy::api::v2::Cluster& clust
       // Otherwise, applyUpdates() will fire with a dangling cluster reference.
       updates_map_.erase(cluster_name);
 
+      // If the active cluster has priority sets and the warming cluster does not have them, it
+      // means that onConfigUpdate is triggered for EDS type of cluster is triggered with no
```

Comment:
Maybe `means that onConfigUpdate was triggered by an EDS update that had no references to this cluster`?